src/index.ts

```diff
@@ -27,13 +29,23 @@ export async function analyze(): Promise<void> {
     '--project',
     PROJECT,
   ];
+  const getTeamArgs = (): string[] => !TEAM ? [] : [
+	  '--team',
+	  TEAM
+  ];
+  const getPolicyArgs = (): string[] => !POLICY ? [] : [
+	  '--policy',
+	  POLICY
+  ];
 
   const getArgs = (cmd: string) => [
     CONTAINER ? 'container' : null,
     cmd,
     ...getEndpointArgs(),
     ...getBranchArgs(),
     ...getProjectArgs(),
+	...getTeamArgs(),
```

Comment:
nit: There are tabs here. Can we change them to spaces?

Reply:
of course :) caefed67